packages/manager/modules/exchange/src/account/mfa/delete/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+import component from './delete.component';
+
+const moduleName = 'ExchangeAccountMfaDelete';
+
+angular
+  .module(moduleName, [])
```

Comment:
Don't you use translations, ui kit or routing ? 🤔

Reply:
Of course ;) I'll add them.